postgres/parser/sem/tree/alter_role.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Dolthub, Inc.
```

Comment:
These should be 2024 now. I'm assuming you copied the headers from the original file, but we may as well update the new files while we're at it. We can leave the other headers as-is though, since these portions are all derived from the Cockroach codebase.